README.md

```diff
@@ -72,10 +82,10 @@ async def test(ctx, *, a):
     return {"x": a}
 
 async def cron(ctx):
-  print("i am a cron job")
+    print("i am a cron job")
 
 async def startup(ctx):
-    ctx["db"] = await create_db()
+    ctx["db"] = DBHelper()
```

Comment:
let's make it async

Reply:
Updated the branch, feel free to modify it to your own beliefs. As long it is a working example I'm happy.